styles.css

```
@@ -70,17 +69,17 @@
   font-size: 1.125rem;
   font-style: normal;
   font-weight: 400;
-  line-height: normal;
+  line-height: 1.6;
 }
 
 .alter-text {
   /* Altered Body Text */
   color: var(--text-color);
   font-family: 'Be Vietnam Pro';
```

Stylelint (17.6.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)